VPLAY-10567:[AAMP] Profile playback stop operation

varshnie opened the pull request:
Reason for change:Profile playback stop operation
Test Procedure: Refer jira ticket
Priority: P1

Vinish100 reviewed (commented):
Looks good to me 👍

Vinish100 commented on AampProfiler.cpp:
```diff
@@ -644,3 +657,29 @@ void ProfileEventAAMP::GetTelemetryParam()
+void ProfileEventAAMP::LogStopTime(const char* streamType)
+{
+    // Print bucket name and values for all relevant profiler buckets
+	AAMPLOG_WARN("IP_AAMP_STOP_TIME: %d,%s,%lld,%s," // version, build, tuneStoptBaseUTCMS ,streamType
```
Please update this in Readme.md

Vinish100 commented on AampProfiler.cpp:
```diff
@@ -644,3 +657,29 @@ void ProfileEventAAMP::GetTelemetryParam()
+				 "%d,%d,"							 // streamer_stop: start, total
+				 "%d,%d,"							 // sink_stop: start, total
+				 "%d,%d,"							 // manifest_downloader_release: start, total
+				 "%d,%d,"							 // tsb_stop: start, total
```
make comments similar to bucket names

varshnie committed:
VPLAY-10567:[AAMP] Profile playback stop operation

Reason for change:Profile playback stop operation
Test Procedure: Refer jira ticket
Priority: P1

Signed-off-by: varshnie <varshniblue14@gmail.com>
